resources/js/src/app/mixins/url.js

```diff
@@ -257,6 +257,14 @@ export default {
             return selectedFilters;
         },
 
+        /**
+         * @param attributeValue
+         * @returns {boolean}
+         */
+        isRangeSliderFilter(attributeValue) {
+            return (typeof attributeValue.min !== 'undefined' && typeof attributeValue.max)
```

Comment:
I think you forgot something in the second condition? It is just `typeof attributeValue.max` but you are not checking what it should be :smile:.

Reply:
:man_facepalming: